Add Asana Release Workflow

GioSensation opened the pull request:
**Asana:** https://app.asana.com/0/1198964220583541/1200878329826704/f
**Reviewer:** @brianhall 

## Description
Automates Asana release tasks creation for the various platforms.

brianhall commented on scripts/release.js:
```diff
@@ -0,0 +1,85 @@
+
+    console.info('Asana on. Duplicating template task...')
+
+    const { new_task } = await duplicateTemplateTask()
```
Very minor, but worth passing `templateTaskGid` as a param here? Makes it slightly more flexible, but I don't feel strongly.

brianhall reviewed (approved):
@GioSensation This looks great, just one really minor comment (optional at your discretion). Another question - should we include a breakdown of this in `README.md` or `CONTRIBUTING.md` so folks understand how to release (and roughly what occurs)?

GioSensation commented:
Good points. I have added info on README.md and we're now passing that parameter explicitly. I have also removed the version field from package.json. We weren't using it anyway and it's confusing when it gets out of sync.

I'll go ahead and merge this to test the pipeline. If you have further points on my latest push I can tackle them in a follow-up.
